.github/workflows/pull-request-checks.yaml

```diff
@@ -170,6 +170,90 @@ jobs:
       - name: Run tests
         run: cd build; ctest . -V -L CORE -j2
 
+  check-ubuntu-20_04-cmake-gcc-KNOWNBUG:
+    runs-on: ubuntu-20.04
+    steps:
+      - uses: actions/checkout@v2
+        with:
+          submodules: recursive
+      - name: Fetch dependencies
+        env:
+          # This is needed in addition to -yq to prevent apt-get from asking for
+          # user input
+          DEBIAN_FRONTEND: noninteractive
+        run: |
+          sudo apt-get update
+          sudo apt-get install --no-install-recommends -yq cmake ninja-build gcc g++ maven flex bison libxml2-utils ccache z3
```

Comment:
The new z3 output on version is not in the new check.

Reply:
Done!